docs/language/learn-ql/javascript/type-tracking.rst

```diff
@@ -453,15 +458,15 @@ This is an example of where `data flow configurations <https://help.semmle.com/Q
 When to use type tracking
 -------------------------
 
-Type tracking and data flow configurations are essentally competing solutions to the same
+Type tracking and data flow configurations are essentially competing solutions to the same
```

Comment:
"complementary" might be nicer than "competing" here

Reply:
Rephrased as "different solutions to the same problem".